plugin/src/main/java/io/jenkins/plugins/coverage/metrics/model/FileWithModifiedLines.java

```diff
@@ -0,0 +1,58 @@
+package io.jenkins.plugins.coverage.metrics.model;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+/**
+ * Model class containing data pertaining to files with modified lines of code. Each object contains a relative file
+ * path and a list of {@link ModifiedLinesBlock} objects.
+ */
+@ExportedBean
```

Comment:
Are you planning to use these classes outside of the API?

Otherwise I would recommend to create a new package (restapi, remoting, api, ...) and move all classes there. Then we can move the *Api classes to the new package as well. 

This would allow to make all internal classes package private as well.